imap_processing/cdf/utils.py

```diff
@@ -16,37 +16,36 @@
 logger = logging.getLogger(__name__)
 
 
-def calc_start_time(
-    shcoarse_time: float,
+def convert_met_to_datetime64(
+    met: np.typing.ArrayLike,
     launch_time: Optional[np.datetime64] = imap_processing.launch_time,
 ) -> np.datetime64:
-    """Calculate the datetime64 from the CCSDS secondary header information.
-
-    Since all instrument has SHCOARSE or MET seconds, we need convert it to
-    UTC. Took this from IDEX code.
+    """Convert mission elapsed time (MET) to numpy datetime64.
 
     Parameters
     ----------
-    shcoarse_time: float
+    met: array_like
```

Comment:
I am not familiar with `array_like` type. Does this mean that input can be array or one input?

Reply:
Yes, needs to be usable by `np.asarray`. Maybe @daralynnrhode knows if there is a better type annotation to put here for numpydoc style?